CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Unreleased
 
+* To include the CSS for all components in the gem, you can now do:
```

Comment:
Should this also be in the README?

Reply:
Yeah, but I don't know where yet because it's already so large. I have a branch to rewrite the README, would you mind if I do it there?